crates/nu-cli/src/commands/str_/substring.rs

```diff
@@ -141,6 +141,7 @@ fn action(input: &Value, options: &Substring, tag: impl Into<Tag>) -> Result<Val
                 )
             })?;
 
+            #[allow(clippy::useless_conversion)]
```

Comment:
If you look at [`Substring`](https://github.com/nushell/nushell/blob/0f688d7da7481c78b54b280661930e0912e1b7b4/crates/nu-cli/src/commands/str_/substring.rs#L86) you'll see that `.0` is already an isize, hence the useless conversion.

You can simply change this to 

```rust
let start = options.0;
```

Reply:
Nice, you're right.